app/ilmoweb/static/signup.js

```diff
@@ -20,4 +21,10 @@ function signup(user_id, group_id, lab_name) {
       }
     }
     http.send(JSON.stringify(data));
-  }
\ No newline at end of file
+  }
+
+function update_signed_up_students(group_id) {
```

Comment:
Make sure that this does not increment the value even when the group size is too big and signup fails